cmd/adapter/main.go

```diff
@@ -257,6 +256,12 @@ func main() {
 		return
 	}
 
+	err = kedautil.ConfigureMaxProcs(logger)
+	if err != nil {
+		logger.Error(err, "failed to set max procs")
+		return
```

Comment:
nit: should this also have exit code 1 to be consistent with other components?
```suggestion
		os.Exit(1)
```

Reply:
I noticed that all the errors are simply returned, like: [here](https://github.com/kedacore/keda/blob/main/cmd/adapter/main.go#L257). so I chose not to modify the existing error-handling behavior.

Reply:
sure, just wondering if it makes sense to unify the behavior
https://github.com/kedacore/keda/blob/0a1d8be9296cfc2dd9be20c2cdaebf9fa1c7a010/cmd/operator/main.go#L118-L122
https://github.com/kedacore/keda/blob/0a1d8be9296cfc2dd9be20c2cdaebf9fa1c7a010/cmd/webhooks/main.go#L82-L86

but it's just a cosmetic change, I'm ok with either path

Reply:
I can create a separate PR to adjust all error codes in the main function to `Exit(1)` if you'd like. but, I think this is behind the scope of the current PR